websites/E/Emby/presence.ts

```diff
@@ -284,6 +288,13 @@ const // official website
 
 let presence: Presence, ApiClient: ApiClient;
 
+/**
+ * refreshApiClient - Initializes the ApiClient object
+ */
+async function refreshApiClient(): Promise<void> {
```

Comment:
Function is only used once so seems a bit silly

Reply:
Replaced method call by its body